Olympus2.hebontes/functions/fn_initArsenalCrate.sqf

```diff
@@ -240,6 +240,6 @@ private _fullAction = [
 [_crate, false] call ace_dragging_fnc_setDraggable;
 
 // Add personal arsenal option to default arsenals
-if (_chosen isEqualTo []) then {
+if (_type isEqualTo "full") then {
```

Comment:
`isEqualTo` is case-sensitive, better use `==` in case user inputs `Full` or `FULL`.
```suggestion
if (_type == "full") then {
```
Above in the function as well.